changelog.d/8276.misc

```diff
@@ -0,0 +1 @@
+Do not attempt to upgrade upgrade database schema on worker processes.
```

Comment:
This should probably have a) fewer "upgrade"s and b) talk about systemd docs? Is this not a `.doc` changelog?

Reply:
it's to match #8266.

Reply:
Serves me right for not paying attention in #8266 then :)